dvpOAuth/__init__.py,sha256=upZvgdtcw3dYRvqOtemO7AuJFAY6L-kf9lWt_pKvWWc,162
dvpOAuth/class_OAuth.py,sha256=eo8xIDw-DwQYUHdFfFmA3IGPVwfVx1jFN-bIYEarigU,12203
dvpOAuth/error.py,sha256=Y3BjsevsSGMDVVdn6ztiQMcGNtB6lySkNPoL7y4BT-Q,131
dvpOAuth/util.py,sha256=y-S4zD5BGEWKHVfJKGyy5teIeV2XdWrMXeRubAW54Ek,685
dvpOAuth-0.0.9.dist-info/LICENSE,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
dvpOAuth-0.0.9.dist-info/METADATA,sha256=0AFqeAkju4UHWn2ga-YJl8w8yv-idAgG6RXNhUSG7p8,991
dvpOAuth-0.0.9.dist-info/WHEEL,sha256=2wepM1nk4DS4eFpYrW1TTqPcoGNfHhhO_i5m4cOimbo,92
dvpOAuth-0.0.9.dist-info/top_level.txt,sha256=14UgmlstFHh0UvlWIljRlFO26u2OesXzFC8SsWFZRmw,9
dvpOAuth-0.0.9.dist-info/RECORD,,
